### DISCUSSION TITLE
DA using Anomaly correction

### DISCUSSION THREAD

**Discussion by vibinj** [NASA-LIS/LISF#1253] [Q&A] [answered]
Dear Sir,
I would like to use the "Anomaly correction" method instead of "CDF matching" for the EnKF data assimilation.
I have not seen any test case related to that. What are the steps I have to follow, Please help.

Thanks
Vibin Jose

**Comment by dmocko** [comment]
Hi @vibinj 

Are you asking about the capability added in this pull request?

https://github.com/NASA-LIS/LISF/pull/791

As you can see, this capability was first added to a different branch, and no testcase was made public at the time.

Much of the LIS team is on leave this week, but we will try to share a test case or sample configs when we return.

**Comment by vibinj** [comment]
Dear Sir,
Thanks for your reply. Suppose I am getting a test case or the config files for the "Anomaly Correction" method; that will be helpful for my work. I will wait for it.


Thanks
Vibin Jose

**Comment by dmocko** [comment, marked as answer]
Hi @vibinj 

We are still looking for a testcase.  But luckily, there are minimal changes to the lis.config over using the EnKF DA methodology.

First, change this line in your lis.config to:

`Data assimilation scaling strategy:   "Anomaly scaling"`

You should also run LDT with this line in your ldt.config:

`Temporal resolution of CDFs:         "yearly"`

Then, back to your lis.config, point to the model and observation CDF files (with yearly temporal resolution) created by LDT, and then specify the `CDF read option` for your assimilated dataset to "0" for yearly.

These small changes should get you started with this methodology.

**Comment by vibinj** [comment]
Dear Mocko,

Thanks. I will try as suggested and get back to you if I have any issues.

Regards
Vibin

**Reply by dmocko** [reply]
Hi @vibinj 

Unfortunately, we are unable to share a testcase, as it's for a project from which we cannot share the config or any datasets.

However, after conferring with the DA team, there are no other config changes required other than described above.

A DA team member also shared this:

"This option, should, however be used with some caution. The implicit assumption here is that most of the biases are in the first moment, which may or may not be true. Also, you have to make sure that the retrievals and the land model estimates are in the same units. For example, ASCAT reports soil moisture in wetness units (0-1), whereas LSM soil moisture is in volumetric units. Therefore, you can’t use this method for ASCAT."
